src/erlcloud_ec2.erl

```diff
@@ -2936,6 +2946,168 @@ net_if_params(#ec2_net_if{}=X) ->
+                        (ec2_flow_ids(), aws_config()) -> ok_error(proplist());
+                        (ec2_max_result(), ec2_token()) -> ok_error(proplist(), ec2_token()). 
+describe_flow_logs(FlowIDs, Filter)
+    when is_list(FlowIDs), is_list(Filter) orelse Filter =:= none ->
+    describe_flow_logs(FlowIDs, Filter, default_config());
+describe_flow_logs(FlowIDs, Config)
+    when is_list(FlowIDs), is_record(Config, aws_config) ->
+    describe_flow_logs(FlowIDs, [], Config).
+
+-spec describe_flow_logs(ec2_flow_ids(), filter_list(), aws_config()) -> ok_error(proplist());
+                        (filter_list(), ec2_max_result(), ec2_token()) -> ok_error(proplist(), ec2_token()).
+describe_flow_logs(FlowIDs, Filter, Config)
+    when is_list(FlowIDs), is_list(Filter) orelse Filter =:= none , is_record(Config, aws_config) ->
+   Params = erlcloud_aws:param_list(FlowIDs, "FlowLogId") ++ list_to_ec2_filter(Filter),
+    case ec2_query(Config, "DescribeFlowLogs", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            Flows = extract_results("DescribeFlowLogsResponse", "flowLogSet", fun extract_flow/1, Doc),
+            {ok, Flows};
+        {error, _} = E -> E
+    end;
+describe_flow_logs(Filter, MaxResults, NextToken)
+    when is_list(Filter) orelse Filter =:= none,
+         is_integer(MaxResults),
+         is_list(NextToken) orelse NextToken =:= undefined ->
+    describe_flow_logs(Filter, MaxResults, NextToken, default_config()).
+
+-spec describe_flow_logs(filter_list(), ec2_max_result(), ec2_token(), aws_config())
+    -> ok_error(proplist(), ec2_token()).
+describe_flow_logs(Filter, MaxResults, NextToken, Config)
+    when is_list(Filter) orelse Filter =:= none,
+         is_integer(MaxResults) andalso MaxResults >= ?FLOWS_MR_MIN andalso MaxResults =< ?FLOWS_MR_MAX,
+         is_list(NextToken) orelse NextToken =:= undefined,
+         is_record(Config, aws_config) ->
+    Params = list_to_ec2_filter(Filter) ++ [{"NextToken", NextToken}, {"MaxResults", MaxResults}],
+    case ec2_query(Config, "DescribeFlowLogs", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            Flows = extract_results("DescribeFlowLogsResponse", "flowLogSet", fun extract_flow/1, Doc),
+            NewNextToken = extract_next_token("DescribeFlowLogsResponse", Doc),
+            {ok, Flows, NewNextToken};
+        {error,  _} = E -> E
+    end.
+
+extract_flow(Node) ->
+    [
+        {deliver_logs_error_message, get_text("deliverLogsErrorMessage", Node)},
+        {resource_id, get_text("resourceId", Node)},
+        {deliver_logs_permission_arn, get_text("deliverLogsPermissionArn", Node)},
+        {flow_log_status, get_text("flowLogStatus", Node)},
+        {creation_time, erlcloud_xml:get_time("creationTime", Node)},
+        {log_group_name, get_text("logGroupName", Node)},
+        {traffic_type, get_text("trafficType", Node)},
+        {flow_log_id, get_text("flowLogId", Node)}
```

Comment:
deliverLogsStatus is missing, no? 
http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_FlowLog.html